setup/gcp/scripts/run_forseti.sh

```diff
@@ -27,7 +27,7 @@ fi
 # inventory command
 MODEL_ID=$(/bin/date -u +%Y%m%dT%H%M%S)
 echo "Run inventory creation"
-forseti inventory create --import_as ${MODEL_ID}
```

Comment:
Why are you losing --import_as? In the future we may decouple from the model and want to only create the inventory. I think --import_as or --initialize_model or some other flag should be used instead of just a positional variable. And it should remain optional.

Reply:
Model_id is still optional at this moment. Michael prefers to take out '--import_as' to make the command more user friendly. I can change it back if @mirons-google is ok with it.

Reply:
I agree with Andrew here.  Importing model should be an option as part of the inventory creation.